apps/opik-backend/src/test/java/com/comet/opik/api/resources/utils/AssertionUtils.java

```diff
@@ -43,4 +47,34 @@ public static ExperimentItem assertFeedbackScoresIgnoredFieldsAndSetThemToNull(E
                         .toList())
                 .build();
     }
+
+    public static void assertTraceComment(Comment expected, Comment actual) {
```

Comment:
This class is too generic and will grow quite fast. I believe it would be better to put it in a dedicated class for comments like CommentResourceClient or something like that